src/main/drivers/camera_control.c

```diff
@@ -60,7 +60,11 @@ void pgResetFn_cameraControlConfig(cameraControlConfig_t *cameraControlConfig)
     cameraControlConfig->refVoltage = 330;
     cameraControlConfig->keyDelayMs = 180;
     cameraControlConfig->internalResistance = 470;
-    cameraControlConfig->ioTag = timerioTagGetByUsage(TIM_USE_CAMERA_CONTROL, 0);
+#ifdef CAMERA_CONTROL_PIN
+    cameraControlConfig->ioTag = IO_TAG(CAMERA_CONTROL_PIN);
+#else
```

Comment:
It is possible to use (black) macro magic so that CAMERA_CONTROL_PIN will expand to tag if defined (including NONE), and to NONE when undefined. It may also check that TAG definition is valid.
It may make this code more readable ..

Reply:
ping me if you think it is good idea ..

Reply:
ping @blckmn :point_up:

Reply:
@ledvinap absolutely! You are the king of macros, so happy for you to show how it'd work.

Reply:
https://github.com/betaflight/betaflight/pull/12393